src/components/common/Input.tsx

```diff
@@ -0,0 +1,64 @@
+import styled from "styled-components";
+
+export interface InputProps {
+  label: string;
+  $width: string;
+  placeholder: string;
+  $textPosition: string;
+  type: string;
+  error?: { isValid: boolean; errorMessage: string };
+  handleInput?: (e: any) => void;
+}
+
+export const Input = ({ label, $width, placeholder, $textPosition, type, handleInput, error }: InputProps) => {
+  const { isValid, errorMessage } = error ?? { isValid: true, errorMessage: "" };
```

Comment:
```suggestion
export const Input = ({ label, $width, placeholder, $textPosition, type, handleInput, error = { isValid: true, errorMessage: "" } }: InputProps) => {
```

default value를 이렇게 설정하면 어떨까요?
또한 handleInput에 들어가는 이벤트 타입을 any 말고... 다른 타입으로 좁혀주세요!

Reply:
defaultvalue를 설정하는 방식을 알려주신 방식대로 수정하고 handleInput 타입을 React.ChangeEvent<HTMLInputElement>로 수정했습니다. 
리액트 타입에 익숙치 않아 처음에 any타입으로 설정 후 코드 작성을 하는 경우가 많은데 나중에 리팩토링 하는 걸 누락할 때가 있는 것 같습니다. 다음부터는 좀 더 신경써서 any타입을 체크해보겠습니다.